packages/components/src/dashicon/index.tsx

```diff
@@ -7,12 +7,23 @@
  */
 /** @typedef {import('react').ComponentPropsWithoutRef<'span'> & OwnProps} Props */
```

Comment:
These JSDocs can be removed as well, although we like to move the prop descriptions (e.g. "Icon name", "Size of the icon") to inline JSDocs in the `types.ts` file ([example](https://github.com/WordPress/gutenberg/blob/367c180b384f6656aa6fa5f6926e0c39ab073905/packages/components/src/tools-panel/types.ts#L10-L12)). This way we can see the descriptions in IntelliSense, docgens, etc.

Reply:
We can remove these `@typedef`s, now that the descriptions are moved to the types.ts file 👍